docs/_reference/auth.md

```diff
@@ -250,6 +250,33 @@ methods:
       }
       ```
 
+  - signature: auth.sendPasswordResetEmail(email, actionCodeSettings)
+    id: sendPasswordResetEmail
+    generator: true
+    description: You can send a password reset email to a user.
```

Comment:
Can you replace with "Sends a password reset email to the given email address.", please? :)